src/test/java/de/tum/in/www1/artemis/util/AbstractArtemisIntegrationTest.java

```diff
@@ -139,4 +142,8 @@ public void mockGetRepositoryPathFromRepositoryUrl(String projectPath, VcsReposi
     public void mockGetProjectKeyFromAnyUrl(String projectKey) {
         doReturn(projectKey).when(urlService).getProjectKeyFromRepositoryUrl(any());
     }
+
+    protected <T, E extends Exception> QueryCountAssert<T, E> assertThatDb(ThrowingProducer<T, E> call) {
```

Comment:
Maybe add a Javadoc here as well even though it is not required by the static code analysis.

Reply:
Accidentally resolved without change or reasoning?

Reply:
Added the javadoc now (indeed had accidentally resolved it)